exchanges/binance/binance_websocket.go

```diff
@@ -102,6 +104,15 @@ func (b *Binance) setupOrderbookManager() {
 			// 10 workers for synchronising book
 			b.SynchroniseWebsocketOrderbook()
```

Comment:
`SynchroniseWebsocketOrderbook()` looks like it may require a `b.obm.jobs` drainer and `ShutdownC` as the first `case`. While `b.obm.jobs` does get reset on a new connection, the channel won't be drained in the event of a websocket shutdown.